src/vivarium_gates_nutrition_optimization_child/data/loader.py

```diff
@@ -1260,7 +1260,7 @@ def load_sqlns_risk_ratios(key: str, location: str) -> pd.DataFrame:
     index_cols = ["age_start", "age_end", "affected_outcome"]
     draw_columns = pd.DataFrame(draw for draw in draws).T
     draw_columns.columns = metadata.ARTIFACT_COLUMNS
-    data = pd.concat([risk_ratios[index_cols], draw_columns], axis=1)
+    data = pd.concat([risk_ratios[index_cols], draw_columns], axis=1).set_index(index_cols)
```

Comment:
Copied from description: "Define index of data in loader function. This should have made it in the artifact PR but it somehow didn't. The artifact I was testing model 7 with had this index update but when I remade this key using the model_7_develop there was no index. I'm adding it here to avoid opening a new PR."